.github/workflows/windows.yml

```diff
@@ -59,6 +59,7 @@ jobs:
     - name: test library
       shell: bash
       run: |
+        cmake --build build --parallel 2 --target setup_tests_a-framework
         cmake --build build --parallel 2 --target test
```

Comment:
Did you meant to restrict tests to just `a-framework`?

Reply:
@bangerth The `test` target will automatically set up quick tests so the line above will run `a-framework/` and `quick_tests/`. Which is more than before.

Reply:
Oh, I misread the patch. I thought you were replacing a line. But you're just adding a line.